packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/editInfo.ts

```diff
@@ -46,6 +46,7 @@ function getInitialEditInfo(image: HTMLImageElement): ImageEditInfo {
         heightPx: image.clientHeight,
         naturalWidth: image.naturalWidth,
         naturalHeight: image.naturalHeight,
+        wasResized: false,
```

Comment:
set to true if image has width or height. Including:
img.width
img.height
img.style.width
img.style.height
If one of them has meaningful value (not initial, not auto, ...), we should treat it as resized